src/common/predictive-text-studio-worker.ts

```diff
@@ -60,4 +64,15 @@ export interface PredictiveTextStudioWorker {
    * Retrieving Keyman keyboard data from the IndexedDB storage
    */
   getDataFromStorage(): Promise<KeyboardDataWithTime[]>;
+
+  /**
+   * Save the KMP package once it compiled
+   * @param KMPFile
+   */
+  saveKMPPackage(KMP: ArrayBuffer): Promise<void>;
+
```

Comment:
It seems like nothing outside of `PredictiveTextStudioWorkerImpl` uses this method, so I would not expose it as part of the public API.

```suggestion
```